ocs_ci/ocs/ocp.py

```diff
@@ -106,6 +106,34 @@ def exec_oc_cmd(self, command, out_yaml_format=True, secrets=None, **kwargs):
             return yaml.safe_load(out)
         return out
 
+    def exec_oc_debug_cmd(self, node, cmd_list):
+        """
+        Function to execute "oc debug" command on OCP node
+
+        Args:
+            node (str): Node name where the command to be executed
+            cmd_list (list): List of commands eg: ['cmd1', 'cmd2']
+
+        Returns:
+            out (str): Returns output of the executed command/commands
+
+        Raises:
+            CommandFailed: When failure in command execution
+        """
+        # Appending one empty value in list for string manipulation
+        cmd_list.append(' ')
+        cmd = " || echo 'cmd failed';".join(cmd_list)
```

Comment:
store str `cmd failed` in some variable like
err_msg = 'CMD FAILED' 
as ^ will be also more visible when debugging and use it when formatting cmd

Reply:
ack